app/views/games/show.html.erb

```diff
@@ -3,14 +3,15 @@
     <section>
       <div><%= image_tag  src="https://cdn11.bigcommerce.com/s-ymgqt/products/24049/images/27742/mario_kart__77494.1432323546.500.750.jpg?c=2", :class => "style_image" %></div>
       <div> <%= button_to  "Favorite ❤️", "/favorties" %> </div>
+      <button><%= link_to 'Write a Review', "/games/#{@game.id}/review" %></button>
     </section>
     <section>
       <div>
         <h2><%= "Title of game" %></h2>
         <p><%= "description of game"%></p>
       </div>
       <div>
-        <p><%= "5 Stars"%></p>
+        <p>Average User Rating: <%= @game.average_user_rating.round(2) %></p>
```

Comment:
I think Mariana was working on the show page display so these elements can move around later on.